src/interactive/azext_interactive/azclishell/app.py

```diff
@@ -205,6 +216,14 @@ def restart_completer(self):
             self.lexer = get_az_lexer(command_info)
         self._cli = None
 
+    def on_recommendation_prepared(self):
```

Comment:
Please refine the method name
```suggestion
    def redraw_scenario_recommendation_info(self):
```